tuneup.py

```diff
@@ -28,6 +37,7 @@ def is_duplicate(title, movies):
     return False
 
 
+
```

Comment:
For part C of this assignment, need to improve this function so it does not take so much time.  In order to do that you have to get the `profile` decorator function working, and then decorate this function with it -- not the main() func